src/cascadia/TerminalApp/TabPaletteItem.cpp

```diff
@@ -19,27 +19,9 @@ using namespace winrt::Microsoft::Terminal::Settings::Model;
 namespace winrt::TerminalApp::implementation
 {
     TabPaletteItem::TabPaletteItem(winrt::TerminalApp::TabBase const& tab) :
-        _Tab(tab)
+        _tab(tab)
     {
         Name(tab.Title());
         Icon(tab.Icon());
-
-        _tabChangedRevoker = tab.PropertyChanged(winrt::auto_revoke, [weakThis{ get_weak() }](auto& sender, auto& e) {
-            auto item{ weakThis.get() };
-            auto senderTab{ sender.try_as<TabBase>() };
-
-            if (item && senderTab)
-            {
-                auto changedProperty = e.PropertyName();
-                if (changedProperty == L"Title")
-                {
-                    item->Name(senderTab.Title());
-                }
-                else if (changedProperty == L"Icon")
-                {
-                    item->Icon(senderTab.Icon());
-                }
-            }
-        });

```

Comment:
are we going to be upset that we don't have this?

Reply:
@DHowett - happy new year! Missed you! 😊 

Yes, we will be a upset a bit, as if the switcher is already visible the change in the tab's name / icon won't get reflected immediately. The update will become visible only the next time the tab switcher is open.

We definitely need to find a solution for this, but I assumed this one is urgent enough to restore the functionality in some follow up.